workflows_api/runtime/src/collection_publisher.py

```diff
@@ -27,8 +25,8 @@ def ingest(self, collection: DashboardCollection, token: str, ingest_api: str):
             "Authorization": f"Bearer {token}",
             "Content-Type": "application/json",
         }
-        response = requests.post(url, data=collection.json(by_alias=True), headers=headers)
-        if response.status_code == 200:
+        response = requests.post(url, data=collection.json(by_alias=True, exclude_unset=True, exclude_none=True), headers=headers)
```

Comment:
`exclude_unset` didn't work for `stac_extensions` - had to add `exclude_none` too